src/content/ui-api/kubectl-gs/gitops/add-enc.md

```diff
@@ -17,12 +15,15 @@ user_questions:
 
 This command configures encryption for the GitOps repository.
 
-Other command this commad depends on:
-- [gitops init]({{< relref "/ui-api/kubectl-gs/gitops/init" >}})
-- [gitops add mc]({{< relref "/ui-api/kubectl-gs/gitops/add-mc" >}})
-- [gitops add org]({{< relref "/ui-api/kubectl-gs/gitops/add-org" >}})
-- [gitops add wc]({{< relref "/ui-api/kubectl-gs/gitops/add-wc" >}})
-- [gitops add app]({{< relref "/ui-api/kubectl-gs/gitops/add-app" >}})
+## Prerequisites
+
+Your GitOps repository should provide the following structural layers:
+
+- Basic structure (see [`init`]({{< relref "/ui-api/kubectl-gs/gitops/init" >}}))
+- Management cluster (see [`add management-cluster`]({{< relref "/ui-api/kubectl-gs/gitops/add-mc" >}}))
+- Organization (see [`add organization`]({{< relref "/ui-api/kubectl-gs/gitops/add-org" >}}))
+- Workload cluster (see [`add workload-cluster`]({{< relref "/ui-api/kubectl-gs/gitops/add-wc" >}}))
+- Apps (see [`add app`]({{< relref "/ui-api/kubectl-gs/gitops/add-app" >}}))
```

Comment:
I think you can run encryption command without having apps, though the goal of the encryption is encrypt app secrets